tests/Fixer/ClassNotation/PhpdocReadonlyClassCommentToKeywordFixerTest.php

```diff
@@ -27,6 +27,8 @@ final class PhpdocReadonlyClassCommentToKeywordFixerTest extends AbstractFixerTe
 {
     /**
      * @dataProvider provideFixCases
+     *
+     * @requires PHP 8.2
```

Comment:
FYI - you can also use the annotation for whole class, not only for single test method.

(non blocker, hint for future)